Integrations/integration-ReversingLabsTitaniumCloud.yml

```diff
@@ -0,0 +1,221 @@
+description: 'ReversingLabs Data provides malware status of the sample. Status can
+  be: malicious, suspicious, known and unknown. Service supports single or bulk queries,
+  with extended option for response data (trust factor, threat level, ...).'
+configuration:
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Base URL
+  name: base
+  defaultvalue: https://ticloud-cdn-api.reversinglabs.com
+  type: 0
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |-
+    import base64
+    import requests
+    from requests.auth import HTTPBasicAuth
+    import re
+    import os
+
+    base_url = demisto.params()['base']
+    user = demisto.params()['credentials']['identifier']
+    password = demisto.params()['credentials']['password']
+    auth = HTTPBasicAuth(user,password)
+
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+
+    def rldata(hash_value):
```

Comment:
comment/function description which describe what are the steps happening in the function could help a lot to faster and easily understand the code.